+Base/QRconv.m

```diff
@@ -0,0 +1,384 @@
+function [v, V, options_fit, stages] = QRconv(img, options_guess, QR_parameters)
+    % QRconv finds QRs in an image via a convoluation algorithm.
+    % img is a NxM image.
+    % options_guess is a struct of the same format as options_fit: struct('ang', ..., 'calibration', ..., 'X_expected', ..., 'Y_expected', ...);
+    % v and V are arrays of column vectors (2xN) of the same size:
+    %  - v are positions of candidate QR codes in the coordinate system of the image img, while 
+    %  - V are the corresponding positons in QR space. 
+    % If a candidate does not pass the checksum, [NaN; NaN] is returned for V.
+    % Also returns transform from QR-space V to position-space v according to 
+    %   v = M * V + b
+    
+    % Default QR parameters.
+    if nargin < 3
```

Comment:
Is it possible to have a default options_guess, or is that just so likely to fail that it's pointless?

Reply:
So likely to fail that it's pointless.